.env

```diff
@@ -1,3 +1,3 @@
-TOKEN=NzA3MjUzNDM4NTM4NTc5OTk5.XrGHQg.b6Wbc3CblHl3UhEZghsz4-ovCzg
+TOKEN=XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
```

Comment:
Rebase and squash to purge history

Reply:
Discord nähtavasti skännib netti ja kohe kui ma pushisin nad resettisid mu tokeni ära.